templates/cluster/vsphere-standalone-cp/templates/vspheremachinetemplate-worker.yaml

```diff
@@ -17,8 +17,9 @@ spec:
             networkName: {{ .Values.worker.network }}
           {{- if .Values.ipamEnabled  }}
             addressesFromPools:
-             - {{- toYaml .Values.ipPool.data | nindent 14 }}
+             - {{- toYaml .Values.ipPool.config | nindent 14 }}
           {{- end }}
+            nameservers: {{ .Values.nameservers }}
```

Comment:
Side question: if `nameservers` aren't set, `nameservers` then would be `nameservers: null` (or maybe `[]`), is it the same as not having the value set at all or should we wrap it like (not sure if the syntax is correct):

```yaml
{{- with .Values.nameservers }}
nameservers: {{ . }}
{{- end }}
```